packages/web/integrations/tfm/router.ts

```diff
@@ -59,18 +59,22 @@ export class TfmRemoteRouter implements TokenOutGivenInRouter {
         priceImpactTokenOut,
       };
     } catch (e) {
+      // perhaps this is from the apiClient, i.e. a timeout
```

Comment:
We should add some unit tests for these routers. We can add this on another PR. I don't want to block this significant performance improvement.

Reply:
Like mocking the response and validating the output? Sounds good. These were originally simple query clients but there is some logic added.